### PR TITLE
Obelisk Books

### PR DESCRIPTION
**Pull request by carlopault**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: 5 new recipes for the curator's various obelisks (most of them were already created, I just forgot to add them to the obelisk code)
add: The curator's obelisks can now be used once per obelisk to create 1 use books that teach everything of the respective tier. This means other people can learn how to make the obelisks now (without buying another book from the Spacestar Ordering Book Catalog).
balance: CODEWORD Arabian now only has a 50% chance of dropping an improvised glock.
balance: Nerfed the health of the curator's obelisks.
fix: Fixed the d100 recipe spawning a d00 die.
soundadd: The obelisks now have sound effects. 
imageadd: Added icons for the books. They were sprited by Crunion and Punkinguy. 
spellcheck: Fixed a typo with CODEWORD Arabian's description.
/:cl:

## About The Pull Request
Punkinguy and Crunion sprited for me. Neo helped me code, as always.

## Why It's Good For The Game
Now the curator can spread the knowledge.

### CODE REVIEW

**Comment by MrEpicSax** [comment]
+1

**Comment by Zyzarda** [comment]
+2 good fixes

**Comment by OK-Computer-OK** [comment]
+1 faggot nice work

File: code/datums/components/crafting/crafting.dm [hunk 1]
```diff
@@ -423,41 +423,46 @@
...
-					if(specialmessage3 == 1)
-						to_chat(usr, "<span class='warning'>The accursed knowledge of the Obelisk of Limitless Wisdom has given you new magic crafting recipes!")
-						to_chat(usr, "<span class ='notice'>You have managed to incorporate an ingenious device which grants the obelisk a backdoor directly into the universal network of magic. Just being in the presence of this thing is giving you head spins. But hey, to hell with the side effects! There’s magic to be done!")
-						specialmessage3 = 0
+					usr.mind.teach_crafting_recipe(/datum/crafting_recipe/tier3/portalgun)
+					usr.mind.teach_crafting_recipe(/datum/crafting_recipe/tier3/gravitygun)
+				//	if(specialmessage3 == 1)
+				//		to_chat(usr, "<span class='warning'>The accursed knowledge of the Obelisk of Limitless Wisdom has given you new magic crafting recipes!")
+				//		to_chat(usr, "<span class ='notice'>You have managed to incorporate an ingenious device which grants the obelisk a backdoor directly into the universal network of magic. Just being in the presence of this thing is giving you head spins. But hey, to hell with the side effects! There’s magic to be done!")
+				//		specialmessage3 = 0
```
Comment (yoyobatty) [inline code/datums/components/crafting/crafting.dm]: Why did you comment this out instead of just deleting it?

Comment (carlopault) [reply code/datums/components/crafting/crafting.dm]: in case I want it to come back

Comment (yoyobatty) [reply code/datums/components/crafting/crafting.dm]: github has built in stuff to revert changes like this and it's easy to do, just remove all the comments that are code

Comment (carlopault) [reply code/datums/components/crafting/crafting.dm]: it needs to stay

**Review by yoyobatty — APPROVED** [review]
alright looks good

**Pull request merged**